### PR TITLE
Search

### PR DESCRIPTION
**Pull request by 1forh**
https://trello.com/c/l9CwFAf4
https://trello.com/c/cLWveDaF

### CODE REVIEW

File: web/themes/fire/dist/bundle.css [hunk 1]
```diff
@@ -1,16 +1,12016 @@
-@import url(https://fonts.googleapis.com/css?family=Roboto:300,400,500,700,900&display=swap);/*!
+@import url(https://fonts.googleapis.com/css?family=Roboto:300,400,500,700,900&display=swap);@charset "UTF-8";
```
Comment (tarawren) [inline web/themes/fire/dist/bundle.css]: npm run build

File: web/themes/fire/templates/includes/typeahead/_typeahead.scss [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    z-index: $z-index-typeahead;
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    background-color: #fff;
+    border: 1px solid $color-gray-border;
+    border-radius: 5px;
+  }
+
+  &-result {
```
Comment (tarawren) [inline web/themes/fire/templates/includes/typeahead/_typeahead.scss]: Should this be empty?

File: web/themes/fire/templates/views/site-search/_site-search.scss [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    }
+  }
+
+  &__meta {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin-bottom: 75px;
+
+    &-filters {
```
Comment (tarawren) [inline web/themes/fire/templates/views/site-search/_site-search.scss]: Should this be empty?

File: web/themes/fire/templates/includes/typeahead/_typeahead.scss (hunk 2 above)
Comment (1forh) [reply web/themes/fire/templates/includes/typeahead/_typeahead.scss]: Heyo! I can remove these but I figured you guys would just have to add them back when you go to style this. Let me know if you want me to remove them!

**Pull request merged**